build.gradle

```diff
@@ -18,6 +18,7 @@ buildscript {
     'androidXSlidingPaneLayout': 'androidx.slidingpanelayout:slidingpanelayout:1.0.0',
     'androidXSwipeRefreshLayout': 'androidx.swiperefreshlayout:swiperefreshlayout:1.0.0',
     'androidXViewPager': 'androidx.viewpager:viewpager:1.0.0',
+    'androidXViewPager2': 'androidx.viewpager2:viewpager2:1.0.0-rc01',
```

Comment:
Just a heads up: can't merge this until 1.0.0 proper is released.

Reply:
Thanks a lot, Jake!